cmd/wksctl/apply/apply.go

```diff
@@ -154,6 +154,8 @@ func (a *Applier) initiateCluster(clusterManifestPath, machinesManifestPath stri
 	if err := installer.SetupSeedNode(wksos.SeedNodeParams{
 		PublicIP:             sp.GetMasterPublicAddress(),
 		PrivateIP:            sp.GetMasterPrivateAddress(),
+		ServicesCIDRBlocks:   sp.Cluster.Spec.ClusterNetwork.Services.CIDRBlocks,
+		PodsCIDRBlocks:       sp.Cluster.Spec.ClusterNetwork.Pods.CIDRBlocks,
```

Comment:
I don't see where we are using this variable.  Can you elaborate on its use?

Reply:
Indeed `ServicesCIDRBlocks` is unused, thought of adding it to follow the schema of the cluster.yaml. It would be useful if we want to implement it, but it's not needed in this PR